hw6-2/cycleList.c

```diff
@@ -0,0 +1,196 @@
+#include <stdlib.h>
+#include <stdbool.h>
+#include <stdio.h>

```

Comment:
```suggestion
#include <stdio.h>
#include "cycleList.h"
```
Не всякий компилятор догадается, что .h и .c файлы связаны, если явно это не сказать